tests/console/sysstat.pm

```diff
@@ -66,7 +66,11 @@ sub run {
     validate_script_output "mpstat",     sub { /CPU    %usr   %nice    %sys %iowait    %irq   %soft  %steal  %guest  %gnice   %idle/ };
     validate_script_output "sar -u",     sub { /CPU     %user     %nice   %system   %iowait    %steal     %idle/ };
     validate_script_output "sar -n DEV", sub { /IFACE   rxpck\/s   txpck\/s    rxkB\/s    txkB\/s   rxcmp\/s   txcmp\/s  rxmcst\/s   %ifutil/ };
-    validate_script_output "sar -b",     sub { /tps      rtps      wtps   bread\/s   bwrtn\/s/ };
+    if (is_sle('>=15-SP1') || is_opensuse) {
```

Comment:
I am not sure which sysstat version is in SLE/Leap 15.x.

Reply:
you can check on https://scc.suse.com/packages for SLE (12.0.2) and https://software.opensuse.org/package/sysstat for Leap (12.0.2)

Reply:
So dropped the condition for >= 15.1 for now.
Also run tidy.